ec/ecdatasource/deploymentdatasource/flatteners_integrations_server.go

```diff
@@ -18,84 +18,109 @@
 package deploymentdatasource
 
 import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/tfsdk"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+
 	"github.com/elastic/cloud-sdk-go/pkg/models"
 
 	"github.com/elastic/terraform-provider-ec/ec/internal/util"
 )
 
 // flattenIntegrationsServerResources takes in IntegrationsServer resource models and returns its
 // flattened form.
-func flattenIntegrationsServerResources(in []*models.IntegrationsServerResourceInfo) []interface{} {
-	var result = make([]interface{}, 0, len(in))
+func flattenIntegrationsServerResources(ctx context.Context, in []*models.IntegrationsServerResourceInfo, target interface{}) diag.Diagnostics {
```

Comment:
Should return the model rather than assign to interface pointer.

Reply:
@pascal-hofmann ?